### ISSUE TITLE
CSS of High-contrast to accessibility doesn't work with radiobutton and checkbox

### ISSUE THREAD

**Issue by mila85** [AudithSoftworks/Uniform#205]
Hi to everyone. It was said that jquery.uniform "works in sync with real form elements to ensure accessibility". I'm modifing the css of the site to assign high-contrast color, but the radiobuttons and checkbox disappeared. Does anyone know how to resolve the problem?
